src/lib/sitemap/tutorials-content-fields.ts

```diff
@@ -13,17 +13,9 @@ import { makeSitemapField } from './helpers'
 import { Collection as ClientCollection } from 'lib/learn-client/types'
 
 function getTutorialLandingPaths(): string[] {
-	const activeSlugs = activeProductSlugs.map(
+	return activeProductSlugs.map(
 		(productSlug: ProductSlug) => `${productSlug}/tutorials`
 	)
-	const sectionOptionsWithLandingPage = Object.values(SectionOption).filter(
-		(option: SectionOption) => option !== SectionOption.onboarding
-	)
-	const sectionOptions = sectionOptionsWithLandingPage.map(
-		(slug: SectionOption) => `${slug}`
-	)
-
-	return [...activeSlugs, ...sectionOptions]
```

Comment:
You can see here that `onboarding` is no longer part of the `SectionOption` enum in `src/lib/learn-client/types.ts`